db/migrate/20141223142759_fill_discovery_attribute_sets_for_existing_hosts.rb

```diff
@@ -1,6 +1,6 @@
 class FillDiscoveryAttributeSetsForExistingHosts < ActiveRecord::Migration
   def up
-    Host::Discovered.scoped.each { |host| host.populate_fields_from_facts}
+    Host::Discovered.all.each { |host| host.populate_fields_from_facts}
```

Comment:
Hmmm if need to support both migrations (on Rails 3 -> Rails 3 and also Rails 3 -> Rails 4) then let's do this in a transparent way. Can you make an if-statement and test for Rails version here? What is the background for this change btw?